src/modules/commons/types.ts

```diff
@@ -0,0 +1,25 @@
+// Common types
+
+export const get = {
+	start: (modelName: string) => `${modelName}_GET_MODEL`,
```

Comment:
I prefer using . for more readability but we can discuss about this

 `${modelName}.GET_MODEL` -> post.GET_MODEL instead of post_GET_MODEL